pkg/api/v1alpha1/cluster.go

```diff
@@ -724,9 +723,12 @@ func validateMirrorConfig(clusterConfig *Cluster) error {
 		return fmt.Errorf("registry mirror port %s is invalid, please provide a valid port", clusterConfig.Spec.RegistryMirrorConfiguration.Port)
 	}
 
-	if !features.RegistryMirrorInsecureSkipVerifySupport().IsActive() {
-		if clusterConfig.Spec.RegistryMirrorConfiguration.InsecureSkipVerify && clusterConfig.Spec.DatacenterRef.Kind != SnowDatacenterKind {
-			return errors.New("insecureSkipVerify is only supported for snow provider")
+	if clusterConfig.Spec.RegistryMirrorConfiguration.InsecureSkipVerify {
+		switch clusterConfig.Spec.DatacenterRef.Kind {
```

Comment:
instead of switch statement, can we maintain a supported insecureSkipVerify provider list and if not within the list, return error